.github/workflows/testsuite.yml

```diff
@@ -0,0 +1,33 @@
+# the test suite runs the tests (headless, server+client) for multiple Meteor releases
+name: Test suite
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        meteorRelease:
+          - '--release 2.3'
```

Comment:
We should also do test for 1.12 as that is what we have in `package.js` as a possible version.